planner/core/point_get_plan.go

```diff
@@ -143,6 +143,83 @@ func (p *PointGetPlan) OutputNames() []*types.FieldName {
 	return p.outputNames
 }
 
+// BatchPointGetPlan represents a physical plan which all children are PointGet and
+// all of them reference the same table and use the same `unique key`
+type BatchPointGetPlan struct {
```

Comment:
In `PointGetPlan`, we have fileds like `UnsignedHandle`, `IsTableDual`, `Lock`, `IsForUpdate`. Does this one need to hold them too?

Reply:
The `UnsignedHandle` filed is used in `ExplainInfo`. The other fields don't need because of related scenario have been filtered at the beginning of `tryWhere2BatchPointGet`.